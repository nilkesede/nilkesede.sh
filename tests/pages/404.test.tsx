import Page404 from '../../src/pages'

import {render} from '../utils'

it('renders correctly', () => {
  const {container} = render(<Page404 />)
  expect(container).toMatchSnapshot()
})
