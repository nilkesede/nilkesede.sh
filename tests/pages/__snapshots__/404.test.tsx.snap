// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c2 {
  width: 200px;
  border-radius: 50%;
  box-shadow: 2px 2px 5px rgba(0,0,0,0.5);
}

.c1 {
  max-width: 1140px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 30px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c7 {
  line-height: 1;
  font-size: 25px;
  margin-left: 15px;
  display: inline-block;
  margin-left: 0;
}

.c9 {
  line-height: 1;
  font-size: 25px;
  margin-left: 15px;
  display: inline-block;
}

.c8 {
  font-family: 'fontello',sans-serif;
  font-style: normal;
  font-weight: normal;
}

.c5 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-top: 1px solid rgba(255,255,255,0.1);
  margin-top: 50px;
  padding: 20px 30px;
}

.c6 {
  font-size: 12px;
  line-height: 1;
}

.c4 {
  margin: 0;
}

.c3 {
  color: #ffffff;
  font-size: 32px;
  margin: 30px 0;
}

.c0 {
  width: 100%;
  min-height: 100vh;
  margin: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media (min-width:768px) {
  .c4 {
    font-size: 24px;
  }
}

@media (min-width:768px) {
  .c3 {
    font-size: 42px;
    margin: 50px 0;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <img
        alt="Nil Késede's avatar image"
        class="c2"
        src="/images/me.jpg"
        srcset="/images/me.jpg, /images/me@2x.jpg 2x, /images/me@3x.jpg 3x"
      />
      <h1
        class="c3"
      >
        Hi, I'm Nil Késede!
      </h1>
      <p
        class="c4"
      >
        A Software Developer based in Fortaleza, Brazil, focused on creating and contribute to high availability apps and improve the user experience with modern tech stacks.
        <br />
        <br />
         
        I’m currently working at the Casa Magalhães's Innovation Core, a commercial automation company.
        <br />
        <br />
         
        Feel free to reach me out on social networks, or you can write to me through 
        <a
          class="anchor"
          href="mailto:me@nilkesede.sh"
          rel="noopener noreferrer"
          target="_blank"
          title="email"
        >
          me@nilkesede.sh
        </a>
        .
      </p>
    </div>
    <div
      class="c5"
    >
      <div
        class="c6"
      >
        Nil Késede © 
        2020
      </div>
      <div>
        <a
          class="c7"
          href="https://linkedin.com/in/nilkesede"
          rel="noopener noreferrer"
          target="_blank"
          title="linkedin"
        >
          <i
            class="c8"
          >
            
          </i>
        </a>
        <a
          class="c9"
          href="https://github.com/nilkesede"
          rel="noopener noreferrer"
          target="_blank"
          title="github"
        >
          <i
            class="c8"
          >
            
          </i>
        </a>
        <a
          class="c9"
          href="https://twitter.com/nilkesede"
          rel="noopener noreferrer"
          target="_blank"
          title="twitter"
        >
          <i
            class="c8"
          >
            
          </i>
        </a>
      </div>
    </div>
  </div>
</div>
`;
